src/components/Container/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropTypes } from 'react';
+import Heading from 'components/Heading';
+import Spinner from 'components/Spinner';
+import Error from 'components/Error';
+
+export const AsyncContainer = ({ loading, error, children }) => {
```

Comment:
It's just a way of breaking down into multiple components. No one has to use this even, it's just more readable.